dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1485,6 +1404,13 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                                     Logger.warn(this.getClass(), "Can't import tree- no matching inodes: {parent=" + t.getParent() + ", child=" + t.getChild() +"}");
                                 }
                             });
+                        } 
+                        else if(obj instanceof MultiTree){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)